R/target_market_share.R

```diff
@@ -470,6 +345,104 @@ aggregate_by_loan_id <- function(data) {
     ungroup()
 }
 
+format_output_dataframe <- function(data){
+  data <- data %>%
+    pivot_wider2(
```

Comment:
didn't you delete this function above somewhere?

Reply:
I was just refactoring. I removed the functions above, and bundled them in a different function that explains the purpose. Just trying to make the function body a little more readable/ clear.